LICENSE.md

```diff
@@ -23,4 +24,6 @@
  creator(s) a pint.
  
  3. Code is provided with no warranty. Using somebody else's code and bitching when it goes wrong makes 
- you a DONKEY dick. Fix the problem yourself. A non-dick would submit the fix back or submit a [bug report](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+ you a DONKEY dick. Fix the problem yourself. A non-dick would submit the fix back or submit a [bug report](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html).
+
+4. If you use code, only the biggest dick ever would call it their own. Even just a comment saying "SOURCE: example.com" would be OK. A non-dick would submit the fix back or submit a [bug report](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
```

Comment:
As this is an actual license that may be used in a legal situaton *(I know, boo me)*, `"SOURCE: example.com"` could be taken too literally.

Would be a bit weird if someone did that, but it could be a mega dick that doesn't like the person that wrote the code and wants to avoid crediting them.

Being vague like this may also turn people away from using projects under DBAD (this already happens), as lawyers would be afraid of the person that wrote the code taking the license literally and getting someone in trouble for not putting `SOURCE: example.com`. (That would make them a hypocritic dick)

Reply:
Replacing
```markdown
Even just a comment saying "SOURCE: example.com" would be OK.
```
with
```markdown
Even just a comment saying the link to the website would be OK.
```

Might work, but you may be able to make it sound better (I'm not the best at wording things)

Reply:
Hm, how 'bout 

`Even just a comment giving attribution to where you found the code would be OK.`

Reply:
Done.